staging/src/k8s.io/apiserver/pkg/server/options/recommended.go

```diff
@@ -62,6 +63,7 @@ func NewRecommendedOptions(prefix string, codec runtime.Codec) *RecommendedOptio
 		CoreAPI:                    NewCoreAPIOptions(),
 		ExtraAdmissionInitializers: func(c *server.RecommendedConfig) ([]admission.PluginInitializer, error) { return nil, nil },
 		Admission:                  NewAdmissionOptions(),
+		ProcessInfo:                DefaultProcessInfo(),
```

Comment:
generic apiservers calling NewRecommendedOptions will get this value silently when they pick up 1.13-level code. they should not inherit a config that defaults them to be "kube-apiserver" in "kube-system".